src/components/dialogs/network-modifications/line/modification/line-modification-dialog.jsx

```diff
@@ -98,12 +114,14 @@ const LineModificationDialog = ({
     const [tabIndexesWithError, setTabIndexesWithError] = useState([]);
     const [dataFetchStatus, setDataFetchStatus] = useState(FetchStatus.IDLE);
     const [lineToModify, setLineToModify] = useState(null);
-    const [tabIndex, setTabIndex] = useState(LineCreationDialogTab.CHARACTERISTICS_TAB);
+    const [tabIndex, setTabIndex] = useState(LineModificationDialogTab.CONNECTIVITY_TAB);
     const [isOpenLineTypesCatalogDialog, setOpenLineTypesCatalogDialog] = useState(false);
 
     const emptyFormData = useMemo(
         () => ({
             [EQUIPMENT_NAME]: '',
+            ...getConnectivityWithPositionEmptyFormData(true, CONNECTIVITY_1),
+            ...getConnectivityWithPositionEmptyFormData(true, CONNECTIVITY_2),
```

Comment:
The root of each tab should be coherent to the tab name, i.e. connectivity. If not, onValidationError does not work, although in this modification form, it is not evident to produce a situation in which the tab connectivity not valid and the tab header should be in 'red'.